pom.xml

```diff
@@ -38,17 +38,18 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>1.642.3</jenkins.version>
+        <jenkins.version>2.60</jenkins.version>
```

Comment:
You will also need to patch `Jenkinsfile` to just `buildPlugin()`. And check `InjectedTest`—seems to be failing on CI.